babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -236,6 +236,31 @@ SqlCreateAttribute CreateTableAttributeJournalTable() :
     { return new SqlCreateAttributeJournalTable(id, getPos()); }
 }
 
+SqlCreateAttribute CreateTableAttributeMergeBlockRatio() :
+{
+    MergeBlockRatioModifier modifier = MergeBlockRatioModifier.UNSPECIFIED;
+    int ratio = 1;
+    boolean percent = false;
+}
+{
+    (
+        (
+            <DEFAULT_> { modifier = MergeBlockRatioModifier.DEFAULT; }
```

Comment:
Is the underscore in the token supposed to be there?

Reply:
Yeah <DEFAULT_> maps to the "DEFAULT" token. I think just <DEFAULT> maps to something  else.